unipacker/core.py

```diff
@@ -119,6 +119,8 @@ class UnpackerEngine(object):
     def __init__(self, sample, unpack_path):
         self.sample = sample
         self.unpack_path = unpack_path
+        dumper = sample.unpacker.dumper.brokenimport_dump_file = unpack_path + \
```

Comment:
I couldn't figure out a cleaner way to access the path since the constructor was called somewhere else but not in this class. Otherwise we will need to pass the path to unpacker then to the dumper which seems even more changes.

Reply:
That's fine like that